cmd/backup/main.go

```diff
@@ -11,8 +11,18 @@ func main() {
 	foreground := flag.Bool("foreground", false, "run the tool in the foreground")
 	profile := flag.String("profile", "", "collect runtime metrics and log them periodically on the given cron expression")
 	flag.Parse()
-
+	additionalArgs := flag.Args()
 	c := newCommand()
+
+	if len(additionalArgs) > 0 {
+		switch additionalArgs[0] {
+		case "show-config":
```

Comment:
Somehow I was imagining this to be called `debug`, but I see how this is not the best name. However `show` sounds a bit too passive to me. Maybe `debug-config`?